ZAPD/ZTexture.cpp

```diff
@@ -871,7 +936,7 @@ void ZTexture::SetTlut(ZTexture* nTlut)
 	assert(nTlut->isPalette);
 	tlut = nTlut;
 
-	textureData.SetPalette(tlut->textureData);
+	textureData.SetPalette(tlut->textureData, splitTlut ? 128 : 0);
```

Comment:
What's the meaning of this 128? Seems like a magic number

Reply:
From discord:
They like to have two skyboxes that share the same palette buffer but occupy halves - 128 colors for one skybox and 128 colors for the other.
It's something we already knew, and we figured that the images that were grayscale were using indices 128-255 but the palette was being loaded into slots 0-127
https://discord.com/channels/688807550715560050/793267277859520563/979230621765369987